src/components/SQForm/SQFormReadOnlyField.js

```diff
@@ -12,6 +12,7 @@ function SQFormReadOnlyField({label, name, size = 'auto', muiFieldProps = {}}) {
   return (
     <Grid item sm={size}>
       <TextField
+        id={label.toLowerCase().replaceAll(' ', '-')}
```

Comment:
I believe this is going to be a common string transformation.
Probably worth creating a helper util function

Not sure of the best name but something like 

`id={toLowerCaseAndHyphenate(label)}`

Reply:
`toKebabCase(label)`? I think kebab-case is traditionally all lower case